pkg/packages/imports.go

```diff
@@ -111,10 +111,16 @@ func parseTypeDec(s string) (string, string) {
 	// pkg.Type
 	// *pkg.Type
 	// Get rid of slice and pointer chars.
-	tmp := strings.NewReplacer(
+
+	tmp := s
+	if strings.HasPrefix(s, "map"){
+		tmp = s[strings.Index(s, "]")+1:]
```

Comment:
The map key could actually be something other than `string` so I think we probably need to `map`s at higher level and call `parseTypeDec` twice for each type.